src/exercise/01/01.py

```diff
@@ -0,0 +1,60 @@
+import random
+
+ROCK = 'ROCK'
+PAPER = 'PAPER'
+SCISSORS = 'SCISSORS'
+
+options = {
+  ROCK: {
+    'id': '1',
+    'label': 'Rock'
+  },
+  PAPER: {
+    'id': '2',
+    'label': 'Paper'
+  },
+  SCISSORS: {
+    'id': '3',
+    'label': 'Scissors'
+  }
+}
+
+def setup_chioces():
+  choices = []
+  for option_item in options.keys():
+    choices.append(options[option_item].get('id'))
+  return choices
+
+def start_game(choices):
+  ai = random.choice(choices)
+  user = input('ROCK, PAPER, SCISSORS!!!!\nChoose Your Option...\n\n1. Rock\n2. Paper\n3. Scissors\n')
+  return {
+    'ai': ai,
+    'user': user
+  }
+
+def determine_win(players, choices):
+  user = players.get('user')
+  ai = players.get('ai')
+
+  if user not in choices:
+    print('You must choose either 1, 2, or 3 as an option.')
+    return
+
+  if ai == user:
+    print('Tie!')
```

Comment:
Im not sure `ai == user` will ever happen, since `ai` is a dictionary and `user` is a string.

Reply:
Huh...that actually has worked every time? Did you try running and playing the game?

Reply:
Wait...they are both strings! ai is a radom choice of the `id` field of each option.

Reply:
yeah, my eyes were tricking me.
Looks good!

It is great you are having fun!